features/ship/always_merge/current_branch/undo_warning.feature

```diff
@@ -0,0 +1,25 @@
+Feature: git-town undo prints a warning message for a merge commit
+
+  Background:
+    Given a Git repo with origin
+    And the branches
+      | NAME    | TYPE    | PARENT | LOCATIONS     |
+      | feature | feature | main   | local, origin |
+    And the current branch is "feature"
+    And the commits
+      | BRANCH  | LOCATION      | MESSAGE        |
+      | feature | local, origin | feature commit |
+    And Git setting "git-town.ship-strategy" is "always-merge"
+    When I run "git-town ship" and close the editor
+    # Make sure that "git-town ship" actually worked.
+    # If this fails then the debugging should focus on Background instead of
+    # Scenarios below.
+    Then the current branch is now "main"
```

Comment:
I just merged #4468, which updates the behavior of `I ran` steps to fail if the executed command exits with a non-zero status. Since Git Town reliably signals failure using exit statuses, this enhancement should make it safe to remove this step and comment while maintaining confidence that the preceding command executed successfully and completely. Let me know if you have any concerns or edge cases where this might not apply!

```suggestion
```

Reply:
`and close the editor` is not currently supported, but it isn't hard to rewrite it to `-m "merge message"` instead.

Reply:
I wanted to suggest that anyways since it executes faster.

Reply:
Done. PTAL.